features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/ParserBase.java

```diff
@@ -323,20 +343,19 @@ public void setThreads(int threads) {
                                         .setDistPoller(identity.getId())
                                         .addParam("monitoringSystemId", identity.getId())
                                         .addParam("monitoringSystemLocation", identity.getLocation())
-                                        .setParam("cause", e.getMessage())
+                                        .setParam("cause", Joiner.on('\n').join(corrections))
                                         .setParam("protocol", protocol.name())
                                         .setParam("illegalFlowEventRate", (int) getIllegalFlowEventRate())
                                         .getEvent());
 
-                                LOG.warn("Illegal flow detected from exporter {}", session.getRemoteAddress().getAddress(), e);
+                                for (final String correction : corrections) {
+                                    LOG.warn("Illegal flow detected from exporter {}: \n{}", session.getRemoteAddress().getAddress(), correction);
```

Comment:
Is this supposed to be `LOG` and not `CORRECTION_LOGGER`?

Reply:
This is already rate-limited (as the event is). The `CORRECTION_LOGGER` is a left-over an can be removed.